frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigCard.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, Fragment } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { makeStyles } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import ControlPanel from './ControlPanel';
+import Footer from './Footer';
+import ConfigList from './ConfigList';
+
+const useStyles = makeStyles((theme) => ({
+  main: {
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: '100vh',
+  },
+}));
+
+export default function AlertConfiguration() {
```

Comment:
why is this ConfigCard file the same as the `AlertConfiguration` file

content also seems similar?
```
<Fragment>
      <CssBaseline />
      <NavBar />
      <main className={classes.main}>
        <ControlPanel addConfig={addConfig}/>
        <ConfigList configs={configs}/>
      </main>
      <Footer />
    </Fragment>
```